.github/ISSUE_TEMPLATE/config.yml

```
@@ -1,5 +1,5 @@
 blank_issues_enabled: false
 contact_links:
-  - name: Open a Discussion
+  - name: Propose a language idea or ask a question
     url: https://github.com/dotnet/csharplang/discussions/new
-    about: Please open new proposals as discussions if they are not fully specified.
+    about: Starting with discussion is the way to get traction on a new proposal.
```

```suggestion
    about: Starting with discussion is the way to create a new proposal.
```

"gain traction" feels like a Microsoft-ism. It's not wrong, but feels weird nonetheless. How about this instead?

---

Awesome, and shorter is better!

I'm using Microsoft-isms now, am I? Not sure how I feel about that. 😁